server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/server.js

```diff
@@ -18,18 +18,19 @@
  */
 "use strict";
 
-var serv;
-var tabletResults;
+var detailTable, historyTable, currentTable, resultsTable;
+
+const url = '/rest/tservers/' + window.location.search.split('=')[1];
```

Comment:
Why get the URL this way? I worry if this is compatible with older browsers.

Reply:
This is just the first method that worked for me. I need the hostname for the server and this is one way I figured out how to do it (grab it from the current page url). There is probably a better way to do this. Any ideas?

Reply:
Currently we are using the template `server.ftl` to get the hostname from the server when it generates the page using the freemarker syntax:
<pre>
// Global constant for the page
        serv = '${server}';
</pre>
You should still be able to do that with a constant.

Reply:
I wasn't quite able to get this working without explicitly passing the variable into the function that initializes the DataTables so that's what I ended up doing. Check out [be1fc59](https://github.com/apache/accumulo/pull/2725/commits/be1fc59b182d087970e8739c1563da4bcf62a818) and let me know if that seems like a fine approach to you.